main.js

```diff
@@ -79,7 +71,12 @@ function submitForm(event) {
 }
 
 function validateInput() {
-  if (validateCategory() && validateDescription() && validateMinutes() && validateSeconds()) {
+  validateCategory();
+  validateDescription();
+  validateMinutes();
+  validateSeconds();
+  validateTime();
+  if (validateCategory() && validateDescription() && validateMinutes() && validateSeconds() && validateTime()) {
```

Comment:
Looks like this will show all applicable error messages at once, which is what we wanted. Very cool.